app/services/school_data_importer.rb

```diff
@@ -0,0 +1,70 @@
+require "open-uri"
+require "csv"
+
+class SchoolDataImporter
+  def run
+    download_and_save_csv
+
+    CSV.foreach(csv_file_name, headers: true, encoding: "windows-1251:utf-8") do |row|
+      local_authority = LocalAuthority.find_or_create_by(
+        code: row["LA (code)"],
```

Comment:
By using `fetch` when accessing this hash where we expect the keys to be present, e.g. `row.fetch("LA (code)")` we can fail early with `Key error` instead of getting `nil` and failing lower down.